test/e2e/fixture/scylla/optimized.scyllacluster.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+metadata:
+  generateName: optimized-
+spec:
+  version: 4.6.3
+  agentVersion: 2.6.3
+  developerMode: true
+  datacenter:
+    name: us-east-1
+    racks:
+    - name: us-east-1a
+      members: 1
+      storage:
+        capacity: 100Mi
+      agentResources:
+        limits:
+          cpu: 50m
+          memory: 200Mi
+      resources:
+        limits:
+          cpu: 1
```

Comment:
can we tune with more reasonable limits?

Reply:
also can we possibly to just enforce req==limits in a particular test to avoid duplicit fixtures? or make a helper that will adjust it to guaranteed class. I admit it's a fine line though.